sdk/schemaregistry/azure-schemaregistry/swagger/README.md

```diff
@@ -19,5 +19,5 @@ clear-output-folder: true
 python: true
 add-credential: true
 credential-scopes: "https://eventhubs.azure.net/.default"
-package-version: "1.0.0b1"
+package-version: "1.0.0b2"
```

Comment:
is this the SR package version? would it be 1.0.0b3 since we just released 2?

Reply:
updated